cmd/dlt/accountroles/cmd.go

```diff
@@ -172,8 +172,11 @@ func run(cmd *cobra.Command, _ []string) {
 			if !confirm.Prompt(true, "Delete the account role '%s'?", role) {
 				continue
 			}
-			err := r.AWSClient.DeleteAccountRole(role)
-			if err != nil {
+			policyARN, err := r.AWSClient.DeleteAccountRole(role)
+			if err != nil && strings.Contains(err.Error(), "DeleteConflict: Cannot delete a policy attached to entities.") {
```

Comment:
There _has_ to be a better way than comparing this. The AWS SDK should report the error type and you can compare against something like `aerr.Type() == awserr.ErrorCodeDeleteConflict`.

Worse case scenario, just do `string.Contains(err.Error(), "DeleteConflict")`

Reply:
I thought there would be something better, thanks for that, I'll work on this Monday

Reply:
sure thing! try something like this:
```go
if err != nil {
  aerr, ok := err.(awserr.Error)
  if ok && aerr.Code() == iam.ErrCodeDeleteConflictException {
    r.Reporter.Warnf("Error deleting policy %s. Cannot delete policy attached to other entities.", policyARN)
  } else {
    r.Reporter.Errorf("There was an error deleting the account roles: %s", err)
  }
  continue
}
```